kube-runtime/src/watcher.rs

```diff
@@ -48,23 +45,24 @@ pub enum Event<K> {
     Delete(K),
     /// The watch stream was restarted.
     ///
-    /// If using the `ListWatch` strategy, this is a relist, and indicates that `InitlPage` events will follow.
-    /// If using the `StreamingList` strategy, this event will be followed by `InitApply` events.
+    /// A series of `InitApply` events are expected to follow until all matching objects
+    /// have been listed. This event can be used to prepare a buffer for `InitApply` events.
     Init,
-    /// A page of objects was received during `Init` with the `ListWatch` strategy.
+    /// Received an object received during `Init`
```

Comment:
nit: could remove the second "received" right before "during".

Reply:
indeed, thanks: https://github.com/kube-rs/kube/pull/1504/commits/29d96a3874d0ab481e79ca89c2c37bcec48420b6